### PR TITLE
Refactor project navigation [SCI-2120]

### PR DESCRIPTION
**Pull request by mlorb**
[SCI-2120](https://biosistemika.atlassian.net/browse/SCI-2120)

### CODE REVIEW

File: app/assets/javascripts/sidebar.js.erb [hunk 1]
```diff
@@ -241,6 +241,15 @@ function setupSidebarTree() {
     e.stopPropagation();
     return false;
   });
+
+  // Add bold style to all levels of selected element
+  $(".tree li.active ")
+  .parents('.parent_li[data-parent="candidate"]')
+  .find("> span a")
```
Comment (ZmagoD) [inline app/assets/javascripts/sidebar.js.erb]: is this a typo? ">"

File: app/views/shared/_left_menu_bar.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div id="left-menu-bar" class="menu-bar">
   <ul class="nav">
     <li class="<%= "active" if projects_are_selected? %>">
-      <a id="projects-link" title="<%= t('left_menu_bar.projects') %>" href="<%= projects_path %>">
+      <a id="projects-link" class="toggle-sidebar-link" title="<%= t('left_menu_bar.projects') %>" href="<%= projects_path %>">
```
Comment (ZmagoD) [inline app/views/shared/_left_menu_bar.html.erb]: you can use link_to helper method here

File: app/assets/javascripts/sidebar.js.erb (hunk 1 above)
Comment (mlorb) [reply app/assets/javascripts/sidebar.js.erb]: It's not a typo.

Comment (ZmagoD) [reply app/assets/javascripts/sidebar.js.erb]: :+1:

**Pull request merged**